cv_next/server/api/cvs.ts

```diff
@@ -164,7 +169,10 @@ export async function getPaginatedCvs(
     if (filterOutDeleted) {
       query = query.eq("deleted", false);
     }
-
+    if(checkNames){
+      ids = await queryUserId;
```

Comment:
Is this working?
I think it should be `{ data: ids }`...

Reply:
Oh I see you simply don't extract data from the result; it's preferable to do so. See the other functions. Lets you early-exit if you encounter an error.

Reply:
Yeah I'll fix it to match the error handling pattern